index.bs

```diff
@@ -10809,9 +10809,12 @@ the <code>typeof</code> operator will return "function" when applied to an inter
             shortest argument list of the entries in |S|.
     1.  Perform [=!=] <a abstract-op>DefinePropertyOrThrow</a>(|F|, "<code>length</code>",
         PropertyDescriptor{\[[Value]]: |length|, \[[Writable]]: <emu-val>false</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val>}).
-    1.  Let |proto| be the [=interface prototype object=] of [=interface=] |I|.
+    1.  Let |proto| be the [=interface prototype object=] of [=interface=] |I| in |realm|.
     1.  Perform [=!=] <a abstract-op>DefinePropertyOrThrow</a>(|F|, "<code>prototype</code>",
         PropertyDescriptor{\[[Value]]: |proto|, \[[Writable]]: <emu-val>false</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>false</emu-val>}).
+    1.  [=Define the constants=] of [=interface=] |I| on |F| given |realm|.
```

Comment:
I might name these "install", but define works too.

Reply:
Not addressing this as per your edit.